src/index.html

```diff
@@ -7,8 +7,89 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet"
+    crossorigin="true">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header-logo">
+        <img
+          class="logo_img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
```

Comment:
Add empty lines between multiline sibling blocks of HTML

<img width="1040" alt="Screenshot 2023-10-25 at 13 37 33" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/aa8706d0-5bf6-49b0-b62c-30b20711fef1">